test/extensions/filters/network/http_connection_manager/config_test.cc

```diff
@@ -379,8 +379,8 @@ stat_prefix: router
   EXPECT_CALL(http_tracer_manager_, getOrCreateHttpTracer(Pointee(ProtoEq(inlined_tracing_config))))
       .WillOnce(Return(http_tracer_));
 
-  HttpConnectionManagerConfig config(parseHttpConnectionManagerFromV2Yaml(yaml_string), context_,
-                                     date_provider_, route_config_provider_manager_,
+  HttpConnectionManagerConfig config(parseHttpConnectionManagerFromV2Yaml(yaml_string, false),
```

Comment:
Again, how come we are avoiding boosting for configs that presumably should be updated to v3?

Reply:
This one I still have a question with.

Reply:
Friendly ping on this one; why do we need to allow boosting here? This is the main thing left IMHO.